src/Conversion/ONNXToKrnl/Tensor/Pad.cpp

```diff
@@ -68,9 +68,23 @@ struct ONNXPadOpLowering : public OpConversionPattern<ONNXPadOp> {
       // This way is to avoid using `select` in computing indices as doing for
       // 'edge' and 'reflect' modes.
       Value cValue;
+      bool useConstantIndexZero = false;
```

Comment:
Code as is is ok, code would be more readable and easier if you did the following:
```C++
  SmallVector<Value, 1> loadIndices;
```
and simply adding `constant zero` to it where you currently set `useConstantIndexZero = true` and then calling the load with that `loadIndices` set that is either empty or has one 0 value.

Reply:
Good idea!  Updated as you suggested.  Thanks!